pyproject.toml

```diff
@@ -26,6 +26,9 @@ dependencies = [
     "duckdb>=1.0.0",
     "datasets>=3.0.0",
     "openai>=1.50.0",
+    "opentelemetry-api>=1.28.2",
```

Comment:
can you add these as extras and relax the constraints a little bit?
`pdm add opentelemetry-api -G opentelemetry-api` should work.

Reply:
Totally!